services/order.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+
+
+def create_order(tickets: list, username: str, date: str = None) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+        if date:
+            order.created_at = date
+            order.save()
+        for ticket in tickets:
+            session = MovieSession.objects.get(pk=ticket["movie_session"])
+            Ticket.objects.create(order_id=order.id,
+                                  row=ticket["row"],
+                                  seat=ticket["seat"],
+                                  movie_session=session)
+
+
+def get_orders(username: str = None) -> Order | QuerySet:
+    if username:
+        return Order.objects.filter(user__username=username)
+    return Order.objects.all()

```

Comment:
Remember:`.all()`, `.filter()` returns QuerySet`.get()`, `.delete()`, `.create()` returns model class, `-> User` for. example